platforms/windows/Cargo.toml

```diff
@@ -16,9 +16,13 @@ accesskit = { version = "0.15.0", path = "../../common" }
 accesskit_consumer = { version = "0.23.0", path = "../../consumer" }
 paste = "1.0"
 static_assertions = "1.1.0"
+# Note: this explicit dependency is required because in `windows` 0.56
+# the `implement` macro changed to use the windows_core crate directly.
+# Keep this in sync with `windows` crate version range.
+windows-core = ">=0.54, <=0.57"
 
 [dependencies.windows]
-version = "0.54"
+version = ">=0.54, <=0.57"
```

Comment:
```suggestion
# Support a range of versions in order to avoid duplication of this crate. Make sure to test all
# versions when bumping to a new release, and only increase the minimum when absolutely necessary.
version = ">=0.54, <=0.57"
```

This is copied from my review of https://github.com/GuillaumeGomez/sysinfo/pull/1295. I don't want the reasoning behind this change to be lost.